tests/unit/test_allocator.cpp

```diff
@@ -413,7 +413,7 @@ TYPED_TEST(IndexAllocatorTest, test_hnsw_reclaim_memory) {
     expected_mem_delta +=
         (hnswIndex->labelLookup.bucket_count() - prev_bucket_count) * sizeof(size_t);
     // New blocks allocated
-    expected_mem_delta += 2 * (sizeof(DataBlock) + vecsimAllocationOverhead);
+    expected_mem_delta += 2 * (sizeof(DataBlock) + vecsimAllocationOverhead + hnswIndex->alignment);
```

Comment:
what about the offset overhead for each vector?

Reply:
what overhead? We allocate the blocks in an aligned manner, and then each vector appears right after the previous one.